DipandaAser/tg-bot-storage
# Bot upload a file to telegram

This pull request allow bot to upload file in telegram

Steps to verify:
- Create a .env file by following [these instructions](https://github.com/DipandaAser/tg-bot-storage/tree/bot-upload-a-file-to-telegram#how-to-test) 
- Run `make test`
- Go inside the chat you specify in .env.test and check if there a new file send with the name test.txt as we can see in this image 👇
![image](https://user-images.githubusercontent.com/54457328/167315474-69850bb4-605f-4982-ae22-ffac76a80415.png)


** Screenshots (optional)
![image](https://user-images.githubusercontent.com/54457328/167315402-cdaf22ae-864b-45cf-8ffc-8f58e23ffce8.png)
